src/pages/pl/getting-started.mdx

```diff
@@ -0,0 +1,115 @@
+---
+layout: ~/layouts/MainLayout.astro
+title: Wprowadzenie
+description: Podstawowy wstęp do Astro.
+i18nReady: true
+---
+import Button from '../../components/Button.astro'
+import ContributorList from '../../components/ContributorList.astro'
+import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro'
+
+:::tip[Astro w wersji 2 już jest!]
+**Co nowego w Astro v2?** [Dowiedz się więcej w poście na blogu](https://astro.build/blog/astro-2/).  
+**Gotowy do aktualizacji?** [Skorzystaj z naszego przewodnika po aktualizacji](/pl/guides/upgrade-to/v2/).
+:::
+
+<h2>Czym jest Astro?</h2>
+
+Astro to **wszechstronny framework** do budowania **szybkich,** **koncentrujących się na treści** stron internetowych. 
+
+## Główne cechy
+
+- **Wyspy komponentów:** Nowa architektura internetowa do budowania szybszych stron internetowych.
+- **Server-first API design:** Przenieś kosztowny proces hydracji z urządzeń Twoich użytkowników na serwer.
+- **Zero JavaScript, domyślnie:** Brak zbędnego JavaScript, który spowalnia działanie strony.
```

Comment:
Nie lepiej byłoby:
- **Zero JavaScripta, domyślnie:** Brak zbędnego kodu, który spowalnia działanie strony.

Reply:
To "Javascripta" wygląda troszkę dziwnie i nie brzmi bardzo po polsku